controllers/nmstate_controller.go

```diff
@@ -173,6 +173,7 @@ func (r *NMStateReconciler) applyHandler(instance *nmstatev1beta1.NMState) error
 	data.Data["CARotateInterval"] = ""
 	data.Data["CAOverlapInterval"] = ""
 	data.Data["CertRotateInterval"] = ""
+	data.Data["CertOverlapInterval"] = ""
```

Comment:
Could you please make the commit message more specific? I mean, instead of `the overlap value` let's use `CertOverlapInterval` so it's clear what overlap you're talking about